### PR TITLE
Create pdf for cheque, user based menu

### PR DESCRIPTION
**Pull request by Chandana-tvc**

### CODE REVIEW

File: user_home/views.py [hunk 1]
```diff
@@ -16,9 +16,15 @@
...
-        return render(request, 'user_homepage.html', {'username':request.user.username})
+    if request.user.is_authenticated and request.user.is_active and profile_instance.flag == 1:
+        if profile_instance.privilege_id.user_type == "Customer":
+            return render(request, 'customer_homepage.html', {'username':request.user.username})
+        elif profile_instance.privilege_id.user_type == "TIER1":
+            return render(request, 'tier1_homepage.html', {'username':request.user.username})
+        elif profile_instance.privilege_id.user_type == "TIER2":
+            return render(request, 'tier2_homepage.html', {'username': request.user.username})
+        elif profile_instance.privilege_id.user_type == "TIER3":
+            return render(request, 'tier3_homepage.html', {'username': request.user.username})
```
Comment (vsridh) [inline user_home/views.py:27]: We have made some changes to privilege IDs.

"TIER1" is now "Tier_1"
"TIER2" is now "Tier_2"
"TIER3" is now "Admin"

File: user_home/templates/tier1_homepage.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block menu %}
+
+<li class="list-group-item list-group-item-light menu" id="fundTransfer">
+  <a href="http://{{ request.get_host }}/internal_user/initFundDeposit">Initiate fund deposit</a> </li>
+<li class="list-group-item list-group-item-light menu" id="issueCheque">
+  <a href="http://{{ request.get_host }}/internal_user/initIssueCheque">Issue cashier cheque</a> </li>
```
Comment (vijaiharihar) [inline user_home/templates/tier1_homepage.html:8]: for both tier 1 and tier 2, have one more option named "Pending requests" that links to "internal_user/"
This gives list of all applicable requests such as transactions, profile update and so on.

File: user_home/templates/customer_homepage.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block menu %}
+<li class="list-group-item list-group-item-light menu" id="makeAppointment">
+  <a href="http://{{ request.get_host }}/user_home/appointment"> Make Appointment </a>
+</li>
+<li class="list-group-item list-group-item-light menu" id="updateProfile">
+  <a href="http://{{ request.get_host }}/user_home/profile_update/"> Update Profile</a>
+</li>
```
Comment (vijaiharihar) [inline user_home/templates/customer_homepage.html:9]: what about transactions here?

File: user_home/views.py (hunk 1 above)
Comment (vijaiharihar) [reply user_home/views.py:27]: Yes. Please use constants in settings.py

File: user_home/templates/customer_homepage.html (hunk 3 above)
Comment (Chandana-tvc) [reply user_home/templates/customer_homepage.html:9]: I am not handling anything realted to customer

File: user_home/templates/tier1_homepage.html (hunk 2 above)
Comment (Chandana-tvc) [reply user_home/templates/tier1_homepage.html:8]: Pending requests is not available in urls.py Ill add place holder, people have to update it later accordingly

**Pull request merged**